### PR TITLE
RES: resolve methods of derived traits

### PR DESCRIPTION
**Pull request by Undin**

### CODE REVIEW

File: src/main/grammars/RustParser.bnf [hunk 1]
```diff
@@ -113,17 +113,32 @@ File ::= [ SHEBANG_LINE ] InnerAttr* Items
 // Attributes
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
-fake Attr ::= MetaItem
+fake Attr ::= MetaItem {
+  extends = "org.rust.lang.core.psi.ext.RsStubbedElementImpl<?>"
+  stubClass = "org.rust.lang.core.stubs.RsPlaceholderStub"
+}
```
Comment (matklad) [inline src/main/grammars/RustParser.bnf]: Yep, stubbing all attributes seems like the right approach to me! And looks like we can now remove `val langAttribute: String?` from `RsTraitItemStub`, and `val pathAttribute: String?` from `RsModDeclItemStub`.

Comment (Undin) [inline src/main/grammars/RustParser.bnf]: You want to save all `RsMetaItem` elements (identifier and literal value) into stub and use them, don't you?

Comment (matklad) [inline src/main/grammars/RustParser.bnf]: Hm, yeah, perhaps we don't want to create stub nodes for all different Literal elements. I think we can store value as a `String?` for now!

Comment (Undin) [inline src/main/grammars/RustParser.bnf]: Done

**Comment by matklad** [comment]
bors r+

Thanks!

**Pull request merged**